config/settings/development.yml

```diff
@@ -2,7 +2,8 @@ PROFILER:
   RACKMINI_ENABLED: true
 
 # URLs
-FEDORA_URL:   'http://fedoraAdmin:fedoraAdmin@localhost:8983/fedora'
+DOR_INDEXING_URL: 'http://localhost:4000/dor'
```

Comment:
Is this the correct port? Or am I looking at the wrong spot here:

https://github.com/sul-dlss/argo/blob/master/.travis.yml#L18

Reply:
Yes, those are different things. 3001 is the workflow service.

Reply:
different service dependency 🙂 

https://github.com/sul-dlss/argo/#run-the-servers